synapse/config/cache.py

```diff
@@ -37,6 +41,20 @@ def __init__(self):
 properties = CacheProperties()
 
 
+def _canonicalise_cache_name(cache_name: str) -> str:
+    """Gets the canonical form of the cache name.
+
+    Since we specify cache names in config and environment variables we need to
+    ignore case and special characters. For example, some caches have asterisks
+    in their name to donate that they're not attached to a particular database
```

Comment:
```suggestion
    in their name to denote that they're not attached to a particular database
```

Reply:
Whoops, didn't see this before merge. I fixed it directly.